crates/core/tedge/src/cli/certificate/cli.rs

```diff
@@ -8,11 +8,9 @@ use super::upload::*;
 use anyhow::anyhow;
 use camino::Utf8PathBuf;
 use clap::ValueHint;
-use tedge_config::explicit_device_id;
```

Comment:
`tedge_config::explicit_device_id` can be fully removed as it was only used here.

Reply:
removed in ef49eeb23438484e78e27beffeba18a9e9e950b5